tests/label/test_polyp_classifications.py

```
@@ -0,0 +1,59 @@
+from django.test import TestCase
+from logging import getLogger
+
+from endoreg_db.models import (
+    Label,
```

Ruff (0.8.2)

`endoreg_db.models.Label` imported but unused

Remove unused import: `endoreg_db.models.Label`

(F401)